### ISSUE TITLE
Is checkbox supported?

### ISSUE THREAD

**Issue by jesobreira** [kikinteractive/app#115]
I develop my apps with App.js, but checkbox seems not to be supported.

When I try using app-input on a checkbox, only its label is displayed (as if it was just a text), and nothing happens on touch.

Is it supposed to support checkbox? Some way to create a toggle just with app.js?

**Comment by amtgard** [comment]
It is not supported.  I rolled a css-only solution for at least Android/Chrome, which you can see here:

https://github.com/amtgard/orkmobile/blob/master/www/css/app.extension.css

It relies on having the Google Material UI Fonts available, and coloring is hard-coded..

**Comment by jesobreira** [comment]
Thank you!

I like this approach. As I needed many other elements, at the time I ended up mixing App.js and MDL from Google.

App.js is a very good framework for simple projects. Sadly it looks dead...

**Comment by amtgard** [comment]
Yes.  I thought this project was a very good idea.  Very simple to jump in
and get work done.  I am thinking about forking it in principal, but
shifting the UI over to more modern tooling.

**Comment by jesobreira** [comment]
Bro, I thought the same thing.
I must take a look on the license to see how far we could go. But I don't think it should bring problems, since this seems to be an abandoned project.
Anyway, if you're gonna create a organization, feel free to invite me :smile:
